### PR TITLE
Fix proxy handling for curl in the remaining charms

### PR DESCRIPTION
**Pull request by dshcherb**
Fix proxy handling for curl in the remaining charms (contrail-agent and contrail-openstack).

### CODE REVIEW

File: contrail-agent/hooks/docker_utils.py [hunk 1]
```diff
@@ -16,13 +20,25 @@
...
                  "software-properties-common"])
-    cmd = ["/bin/bash", "-c",
-           "set -o pipefail ; curl -fsSL --connect-timeout 10 "
-           "https://download.docker.com/linux/ubuntu/gpg "
-           "| sudo apt-key add -"]
+    cmd = [
+        "/bin/bash", "-c",
+        "set -o pipefail ; curl {}"
+        "-fsSL --connect-timeout 10 "
+        "https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -"
```
Comment (Andrey-mp) [inline contrail-agent/hooks/docker_utils.py]: this is already in patch #48 - is it needed here?

File: contrail-agent/hooks/docker_utils.py [hunk 2]
```diff
@@ -16,13 +20,25 @@
...
 def install():
     apt_install(["apt-transport-https", "ca-certificates", "curl",
                  "software-properties-common"])
-    cmd = ["/bin/bash", "-c",
-           "set -o pipefail ; curl -fsSL --connect-timeout 10 "
-           "https://download.docker.com/linux/ubuntu/gpg "
-           "| sudo apt-key add -"]
+    cmd = [
+        "/bin/bash", "-c",
+        "set -o pipefail ; curl {}"
```
Comment (Andrey-mp) [inline contrail-agent/hooks/docker_utils.py]: please add space after {} in all 4 files.

**Comment by Andrey-mp** [comment]
also please update charmhelpers in contrail-keystone-auth charm. charmhelpers must be same version in all contrail charms

File: contrail-agent/hooks/docker_utils.py (hunk 2 above)
Comment (dshcherb) [reply contrail-agent/hooks/docker_utils.py]: Thanks for catching!

File: contrail-agent/hooks/docker_utils.py (hunk 1 above)
Comment (dshcherb) [reply contrail-agent/hooks/docker_utils.py]: Yes this is not needed branched off with that commit included. Rebased now.

**Comment by dshcherb** [comment]
@Andrey-mp Thanks a lot for the review. I rebased the changes.

**Pull request merged**